.travis.yml

```diff
@@ -90,21 +90,73 @@ install:
   - export AWESOME_VERSION="${TRAVIS_BRANCH}-g$(git rev-parse --short HEAD)"
   - 'if [ "$TRAVIS_PULL_REQUEST" != false ]; then AWESOME_VERSION="${AWESOME_VERSION}-PR${TRAVIS_PULL_REQUEST}"; fi'
 
+  # Functions for custom Travis folds.
+  - |
+    travis_fold_start() {
+        echo -n "travis_"
+        echo "fold:start:$1"
+        travis_fold_current="$1"
+    }
+    travis_fold_end() {
+        echo -n "travis_"
+        echo "fold:end:$travis_fold_current"
+    }
 script:
   - export CMAKE_ARGS="-DLUA_LIBRARY=${LUALIBRARY} -DLUA_INCLUDE_DIR=${LUAINCLUDE} -D OVERRIDE_VERSION=$AWESOME_VERSION -D DO_COVERAGE=${DO_COVERAGE}"
   - |
     if [ -n "$BUILD_IN_DIR" ]; then
-        SOURCE_DIRECTORY="$PWD"
-        mkdir "$BUILD_IN_DIR"
-        cd "$BUILD_IN_DIR"
-        cmake $CMAKE_ARGS "$SOURCE_DIRECTORY"
+      travis_fold_start "build_in_dir"
+      SOURCE_DIRECTORY="$PWD"
+      mkdir "$BUILD_IN_DIR"
+      cd "$BUILD_IN_DIR"
+      cmake $CMAKE_ARGS "$SOURCE_DIRECTORY"
+      cd -
+      travis_fold_end
+    fi
+  - |
+    travis_fold_start "make"
+    make
+    travis_fold_end
+    if [ "$TRAVIS_TEST_RESULT" = 0 ]; then
+      travis_fold_start "make.install"
+      sudo env PATH=$PATH make install && awesome --version
+      travis_fold_end
+    fi
+    if [ "$TRAVIS_TEST_RESULT" = 0 ]; then
```

Comment:
This does not work!

`.travis.yml`:
```
script:
        - |
          echo $TRAVIS_TEST_RESULT
          true
          echo $TRAVIS_TEST_RESULT
          false
          echo $TRAVIS_TEST_RESULT
```
Build: https://travis-ci.org/psychon/awesome/builds/135220421
(Relevant part is: `$TRAVIS_TEST_RESULT` always stays 0 and the build is successfull)

So this variable is only updated at the end of each "command block".